heron/tools/apiserver/src/java/com/twitter/heron/apiserver/utils/FileHelper.java

```diff
@@ -110,6 +111,23 @@ private static void addFileToArchive(TarArchiveOutputStream archiveOutputStream,
     }
   }
 
+  // save uploaded file to new location
+  public static void writeToFile(InputStream uploadedInputStream,
+                           String uploadedFileLocation) throws IOException {
```

Comment:
Just to confirm that this indent is expected or not.

Reply:
I will fix the incorrect indent